### PR TITLE
Add primary_script for most Noto fonts

### PR DESCRIPTION
**Pull request by tomasdev**
Included all Noto Fonts in this CL based on these conditions:
- Should have at least one language
- Should support the latin subset
- Primary script is the most repeated script in cases where multiple scripts are supported

Noto Sans, Noto Serif, Noto Display are exceptions because there is no clear primary script in those. Perhaps it should be Latn.

Internal reference b/261708463

### CODE REVIEW

File: ofl/notosansarabicui/METADATA.pb [hunk 1]
```diff
@@ -106,3 +106,4 @@ languages: "wni_Arab"  # Ndzwani Comorian
 languages: "wo_Arab"  # Wolof, Arabic
 languages: "zdj_Arab"  # Ngazidja Comorian
 languages: "zlm_Arab"  # Malay, Arabic
+primary_script: "Arab"
```
Comment (chrissimpkins) [inline ofl/notosansarabicui/METADATA.pb:109]: The UI families are not used in fonts.google.com.  We probably do not need these metadata there?

File: ofl/notosansbengaliui/METADATA.pb [hunk 2]
```diff
@@ -42,3 +42,4 @@ languages: "syl_Beng"  # Sylheti
 languages: "trp_Beng"  # Kok Borok, Bangla
 languages: "unr_Beng"  # Mundari
 languages: "unx_Beng"  # Munda
+primary_script: "Beng"
```
Comment (chrissimpkins) [inline ofl/notosansbengaliui/METADATA.pb:45]: UI family that is not released on fonts.google.com

**Review by chrissimpkins — APPROVED** [review]
LGTM.  There are two "UI" families that are not released through fonts.google.com.  There is no harm in having the metadata listed in these families, but I don't believe that it is necessary in general.

**Comment by chrissimpkins** [comment]
@simoncozens Any concerns about the addition of primary_script metadata across the Noto projects?  If we do this, do we need to update the builder so that it preserves the data as these family are changed going forward?

**Comment by simoncozens** [comment]
No, it’s fine. Saves me a lot of work. :-)

**Pull request merged**